homeassistant/components/media_player/const.py

```diff
@@ -78,6 +79,11 @@
 SERVICE_SELECT_SOUND_MODE = "select_sound_mode"
 SERVICE_SELECT_SOURCE = "select_source"
 
+REPEAT_MODE_ALL = "all"
+REPEAT_MODE_OFF = "off"
+REPEAT_MODE_ONE = "one"
+REPEAT_MODES = [REPEAT_MODE_ALL, REPEAT_MODE_OFF, REPEAT_MODE_ONE]
```

Comment:
When I cycle through modes in my music players, it's usually off, all, 1.

Reply:
Are you saying that I should change `"one"` to `"1"` or that I should sort the list in that order rather than alphabetically?

Or maybe you are just sharing an anecdote? :-)

Reply:
@balloob Sorry to bug you but I am a bit stalled here since I don't know what you mean.

I'll just add that the order of the list is not used for presentation.

Reply:
All right, I will merge as is.

Reply:
I think it's the order of modes that is referenced. We want it to be off, all, one instead of all, off, one.